.gitlab-ci.yml

```diff
@@ -2,7 +2,17 @@ include:
   remote: 'https://raw.githubusercontent.com/ebi-ait/gitlab-ci-templates/master/build-release-deploy.yml'
 
 Test:
-  image: quay.io/ebi-ait/ingest-base-images:trion_ng-cli-karma_11.2.6
+  image: quay.io/ebi-ait/ingest-base-images:node_14.15.5-alpine
+  stage: test
+  before_script:
+    - yarn install --frozen-lockfile --no-progress
+    - apk add chromium
+    - export CHROME_BIN=/usr/bin/chromium-browser
   script:
-    - yarn install
-    - ng lint && ng test --watch false
+    - yarn lint
```

Comment:
Is the bulid job configured to fail when there are (too many) linting problems?
If not, why are we running it during build?

Reply:
I don't know why we lint, I was just replicating the previous linting step.